src/registrar/tests/test_models_domain.py

```diff
@@ -103,15 +110,58 @@ def test_cache_nested_elements(self):
 
         # check contacts
         self.assertEqual(domain._cache["_contacts"], self.mockDataInfoDomain.contacts)
-        self.assertEqual(domain._cache["contacts"], [expectedContactsDict])
+        # The contact list should not contain what is sent by the registry by default,
+        # as _fetch_cache will transform the type to PublicContact
+        self.assertNotEqual(domain._cache["contacts"], expectedUnfurledContactsList)
+        self.assertEqual(domain._cache["contacts"], expectedContactsList)
 
         # get and check hosts is set correctly
         domain._get_property("hosts")
         self.assertEqual(domain._cache["hosts"], [expectedHostsDict])
 
-    def tearDown(self) -> None:
```

Comment:
Why did you remove the teardown?

Reply:
Still exists! Just moved it to the beginning of the test